pkgs/development/web/nodejs/v18.nix

```diff
@@ -19,8 +19,8 @@ let
 in
 buildNodejs {
   inherit enableNpm;
-  version = "18.20.0";
-  sha256 = "sha256-BMhneaLMfu/fzzeanYWIOqHTsdyJCbYiGxY2hIF4VqQ=";
+  version = "18.19.1";
```

Comment:
Would you mind leaving a comment about the `ffi-napi` breakage? 
Besides that, LGTM. Thanks!